package.json

```diff
@@ -30,5 +30,8 @@
   },
   "lint-staged": {
     "**/*": "prettier --write --ignore-unknown"
+  },
+  "dependencies": {
+    "@voucherify/sdk": "^2.5.0"
```

Comment:
It looks like this dependency for the for the voucherify sdk was incorrectly added to the monorepo root package.json? Please remove and ensure it is only installed in the necessary app and package(s)